### PR TITLE
doc(Readme): Centering heading and labelled badges

### PR DESCRIPTION
**Pull request by DEBSUBHRO**
Signed-off-by: DEBSUBHRO <debsubhro.roy10@yahoo.com>

## Issue #79 
 
**Changes**
- Centering the title 
- Centering the badges
 
## Related Issues
 
* Issue #79
* Pull Request #76

### CODE REVIEW

**Comment by DEBSUBHRO** [comment]
> This doesn't seem right. Look at what it is showing as here: https://github.com/DEBSUBHRO/markdown-editor/tree/fix/readme#--markdown-editor

Sorry, Ma'am. I debugged the code.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
-[![downloads](https://img.shields.io/npm/dm/@accordproject/markdown-editor)](https://www.npmjs.com/package/@accordproject/markdown-editor)
-[![GitHub license](https://img.shields.io/github/license/accordproject/markdown-editor?color=bright-green)](https://github.com/accordproject/markdown-editor/blob/master/LICENSE)
-[![npm version](https://badge.fury.io/js/%40accordproject%2Fmarkdown-editor.svg)](https://badge.fury.io/js/%40accordproject%2Fmarkdown-editor)
-[![join slack](https://img.shields.io/badge/Slack-Join%20Slack-blue)](https://accord-project-slack-signup.herokuapp.com/)
+
+
+<p align="center">
+
+
+   <a href="https://app.netlify.com/sites/accordproject-markdown-editor/deploys">
```
Comment (irmerk) [inline README.md]: I believe this could be changed to `https://accordproject-markdown-editor.netlify.com/examples/`

Comment (DEBSUBHRO) [reply README.md]: Yes, ma'am.

> One change on a hyperlink.

Yes, ma'am

Comment (DEBSUBHRO) [reply README.md]: Yes, ma'am.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,26 @@
...
+      <img src="https://img.shields.io/npm/dm/@accordproject/markdown-editor" alt="npm version" />
+   </a>
+ 
+   <a href="https://badge.fury.io/js/%40accordproject%2Fmarkdown-editor">
+      <img src="https://badge.fury.io/js/%40accordproject%2Fmarkdown-editor.svg" alt="downloads" />
+   </a>
+
+   <a href="https://badge.fury.io/js/%40accordproject%2Fmarkdown-editor">
+      <img src="https://img.shields.io/badge/Slack-Join%20Slack-blue" alt="join slack" />
+   </a>
```
Comment (irmerk) [inline README.md]: I'm curious what this `badge.fury.io` link is doing for the actual `a href` here?

Comment (DEBSUBHRO) [reply README.md]: Sorry, ma'am.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,26 @@
...
+
+   <a href="https://accordproject-markdown-editor.netlify.com/examples/">
+      <img src="https://api.netlify.com/api/v1/badges/952fdc5d-a2bb-4895-a375-25ea1c6f30d8/deploy-status" alt="Netlify Status" />
+   </a>
+
+   <a href="https://www.npmjs.com/package/@accordproject/markdown-editor">
+      <img src="https://img.shields.io/npm/dm/@accordproject/markdown-editor" alt="npm version" />
+   </a>
+ 
+   <a href="https://badge.fury.io/js/%40accordproject%2Fmarkdown-editor">
```
Comment (irmerk) [inline README.md]: I think this would need to be changed as well.

Comment (DEBSUBHRO) [reply README.md]: But, ma'am, the link seems to be correct.

If not, then please enlighten me on what should I do next.
Thank you.

File: README.md [hunk 4]
```diff
@@ -1,10 +1,26 @@
...
+      <img src="https://badge.fury.io/js/%40accordproject%2Fmarkdown-editor.svg" alt="downloads" />
+   </a>
+
+   <a href="https://accord-project-slack-signup.herokuapp.com/">
+      <img src="https://img.shields.io/badge/Slack-Join%20Slack-blue" alt="join slack" />
+   </a>
```
Comment (irmerk) [inline README.md]: Are we able to use the badge used in https://github.com/accordproject/techdocs?

Comment (DEBSUBHRO) [reply README.md]: Sure, Ma'am. Changes done. Please check it.

**Pull request merged**